salt/cloud/deploy/bootstrap-salt.sh

```diff
@@ -1304,7 +1304,7 @@ __ubuntu_derivatives_translation() {
     linuxmint_19_ubuntu_base="18.04"
     linaro_12_ubuntu_base="12.04"
     elementary_os_02_ubuntu_base="12.04"
-    neon_16_ubuntu_base="16.04"
+    neon="16.04"
```

Comment:
this should still be `neon_16_ubuntu_base`

Reply:
shouldn't see any updates in bootstrap-salt.sh file unless doing a bootstrap release